routes/login.js

```diff
@@ -1,21 +1,47 @@
 import express from 'express';
-import jwt from 'jsonwebtoken';
 import asyncHandler from './../middlewares/asyncMiddleware.js';
+import {generateToken} from "../jwt/token.js";
+import { pool } from './../dbConfig.js';
+import bcrypt from "bcrypt";
 
 const router = express.Router();
 
-const generateToken = (user) => {
-  const signature = 'my-super-secret';
-  const expiration = '6h';
-  return jwt.sign({ user }, signature, { expiresIn: expiration });
-};
-
 router.post('/', asyncHandler((req, res) => {
-  const user = req.body;
-  const token = generateToken(user);
-  res.json({
-    token: token
-  });
+  const {username, password} = req.body;
+  pool.query('SELECT * FROM public."Users" WHERE username = $1',
+    [username],
+    (error, result) => {
+      if (error) {
+        throw error;
+      }
+      if(result.rows.length > 0){

```

Comment:
Try to avoid callback API at all costs. Do not use raw SQL - it's dangerous and not flexible. Take a look at a library called `sequelize`.